cli/jobs/automl-standalone-jobs/cli-automl-forecasting-task-energy-demand/cli-automl-forecasting-task-energy-demand.yml

```diff
@@ -46,8 +46,8 @@ training:
   enable_model_explainability: true
   enable_stack_ensemble: false
 
-#blocked_training_algorithms:
-#  - prophet
+blocked_training_algorithms:
+  - prophet
```

Comment:
nit: blocked training algorithms here is not going to take any effect (since there's already an allowed training algorithm specified - so all iterations are going to be recommended with auto_arima anyways)
I'd suggest to update the examples such that they mimic the real user scenarios - in which case only one of the two properties is sufficient. (not blocking this pr but feel free to follow up) @CESARDELATORRE FYI

Reply:
Also, do take a look at the error on the gate (might be related to the changes in here)
![image](https://user-images.githubusercontent.com/6081643/165146012-9483e60d-ab7a-4011-a14d-fdcb20c8d4c2.png)